assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -461,16 +471,30 @@ const MidRouteSwingOnSecondHalf = ({
   />
 )
 
-const getSequence = (tripIndex: number, pieceTrips: (Trip | AsDirected)[]) => {
+type Sequence = "first" | "middle" | "last"
+type NonRevenueStatus = "pull-out" | "pull-back" | "deadhead"
```

Comment:
**praise:** I like splitting these out into separate types even if it's a simple mapping, it makes things a bit clearer.